metadoc-js/src/main/scala/metadoc/ScalaDocumentSymbolProvider.scala

```diff
@@ -41,7 +40,8 @@ class ScalaDocumentSymbolProvider(index: MetadocSemanticdbIndex)
         case DocumentSymbol(denotation, kind, definition) =>
           new SymbolInformation(
             name = denotation.name,
-            containerName = denotation.signature,
+            // TODO: pretty print `.tpe`
```

Comment:
What do you mean by pretty printing here?

Reply:
`SymbolInformation.tpe` is a fairly complex proto message https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb3/semanticdb3.md#type this refers to ticket https://github.com/scalameta/scalameta/issues/1479 about making a readable string representation of types. Added link to ticket.